qcodes/instrument/parameter.py

```diff
@@ -928,6 +942,110 @@ def sweep(self, start, stop, step=None, num=None):
                                 step=step, num=num)
 
 
```

Comment:
should `It is not recommended for any new driver.` be bold in `Creating QCoDeS instrument drivers` notebook?